app/views/classified_listings/_form.html.erb

```diff
@@ -55,6 +55,10 @@
       </div>
     </div>
       <%= javascript_pack_tag "listingForm", defer: true %>
+    <div class="field">
+      <%= form.label "location", "Location" %>
+      <%= form.text_field "location", placeholder: "64 characters max, plain text" %>
```

Comment:
this is not validated in the model, so in theory I could add as many chars as I want to, would it break the UI?

Reply:
Good catch! I should probably limit it in the model as well to keep things short. If the location is long it pushes to the edit/report abuse buttons to a new line, so I could look into dedicating a line just for location (if provided of course) as to not mess with the placement of action buttons.